metadata/ota_metadata/metadata_gen.py

```diff
@@ -135,10 +170,21 @@ def gen_metadata(
         ]
         f.writelines("\n".join(symlink_list))
 
+    # compression with zstd
+    #   store the compressed file with its original file's hash as name,
+    #   directly under the <compressed_dir>
+    compression_enabled = False
+    if compressed_dir:
```

Comment:
compression_enable and compressed_dir are having same meaning. I think `compression_enabled` can be removed.